integration-tests/src/test/resources/application.properties

```diff
@@ -1 +1,3 @@
-custom.config.wiremock.url=http://localhost:${quarkus.wiremock.devservices.port}/mock-me
\ No newline at end of file
+custom.config.wiremock.url=http://localhost:${quarkus.wiremock.devservices.port}/mock-me
+quarkus.rest-client."io.quarkiverse.wiremock.devservice.QuarkusClient".enable-local-proxy=true
+quarkus.rest-client."io.quarkiverse.wiremock.devservice.QuarkusClient".url=https://imaginary-host.io/
```

Comment:
the test is failing without this config, but this looks like an issue on quarkus side more than on this extension